src/Symfony/Bundle/WebProfilerBundle/Twig/WebProfilerExtension.php

```diff
@@ -12,31 +12,89 @@
 namespace Symfony\Bundle\WebProfilerBundle\Twig;
 
 use Symfony\Component\HttpKernel\DataCollector\Util\ValueExporter;
+use Symfony\Component\VarDumper\Cloner\Data;
+use Symfony\Component\VarDumper\Dumper\HtmlDumper;
 
 /**
  * Twig extension for the profiler.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class WebProfilerExtension extends \Twig_Extension
+class WebProfilerExtension extends \Twig_Extension_Profiler
```

Comment:
this looks wrong to me, because it will register the twig profiler token parsers twice (and this extension will register them without the Stopwatch integration, which means it is bad if this extension wins)